File: 1 Revalida en python/5 Funciones/3.py
#funcion para saber que es bisiesto
def es_bisiesto(n):
	if(((n%4)==0)and((n%100)!=0)or((n%400)==0)):
		return True
	else:
		return False
#se quiere saber los anios bisiestos de los 2000 primeros anios
n=1
for n in range(1,2000):
	if  (es_bisiesto(n)):
		print (n)

File: 2 Revalida en C++/5 Funciones/3.cpp
///funcion para saber que es bisiesto
#include <bits/stdc++.h>
using namespace std;
int es_bisiesto(int n)
{
    if((((n%4)==0)&&((n%100)!=0))||((n%400)==0))
		return true;
	else
		return false;
}
int main()
{
    for (int n=1;n<=3000;n++)
	{
	    if  (es_bisiesto(n))
		cout<<"los anios bisiestos"<<n<<endl;
	}
    return 0;
}
